lib/Zonemaster/CLI.pm

```diff
@@ -623,6 +623,10 @@ sub run {
         if ( $self->test and @{ $self->test } > 0 ) {
             foreach my $t ( @{ $self->test } ) {
                 my ( $module, $method ) = split( '/', lc($t), 2 );
+                if ( $module =~ /^([a-z]+)[0-9]+$/ ) {
+                    $method = $module;
+                    $module = $1;
+                }
```

Comment:
If $module matches a test case identifier then $method can be any garbage. I guess that is OK in this context.

Reply:
Good idea. Currently the message comes from the Engine which has no idea of the `--list-tests` CLI option. I guess it would mean rethinking how this message/error is emitted.

> But that’s a problem best addressed in another PR.

I'll open an issue with your suggestion.